applications/minotari_ledger_wallet/common/src/common_types.rs

```diff
@@ -97,42 +102,61 @@ impl Instruction {
 
 /// Key manager branches shared by the Ledger application and the wallet.
 #[repr(u8)]
-#[derive(Debug, Copy, Clone, PartialEq)]
-pub enum Branch {
-    DataEncryption = 0x00,
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Serialize, Deserialize)]
+pub enum LedgerKeyBranch {
     MetadataEphemeralNonce = 0x01,
-    CommitmentMask = 0x02,
-    Nonce = 0x03,
-    KernelNonce = 0x04,
-    SenderOffset = 0x05,
     OneSidedSenderOffset = 0x06,
-    Spend = 0x07,
-    RandomKey = 0x08,
+    Random = 0x08,
     PreMine = 0x09,
-    CodeTemplateAuthor = 0x0a,
+    Spend = 0x07,
 }
 
-impl Branch {
+impl LedgerKeyBranch {
     pub fn as_byte(self) -> u8 {
         self as u8
     }
 
     pub fn from_byte(value: u8) -> Option<Self> {
         match value {
-            0x00 => Some(Branch::DataEncryption),
-            0x01 => Some(Branch::MetadataEphemeralNonce),
-            0x02 => Some(Branch::CommitmentMask),
-            0x03 => Some(Branch::Nonce),
-            0x04 => Some(Branch::KernelNonce),
-            0x05 => Some(Branch::SenderOffset),
-            0x06 => Some(Branch::OneSidedSenderOffset),
-            0x07 => Some(Branch::Spend),
-            0x08 => Some(Branch::RandomKey),
-            0x09 => Some(Branch::PreMine),
-            0x0a => Some(Branch::CodeTemplateAuthor),
+            0x01 => Some(LedgerKeyBranch::MetadataEphemeralNonce),
```

Comment:
I think we should keep the old values here, as comments perhaps?

Reply:
the ones removed where never used by the ledger.